clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -1516,10 +1516,9 @@ protected static void checkAndHandleUpstreamOffsetRewind(
        * otherwise, don't fail the push job, it's streaming ingestion now so it's serving online traffic already.
        */
       String logMsg = String.format(
-          "Replica: {} received message at offset: {} with upstreamOffset: {};"
-              + " but recorded upstreamOffset is: {}. Received message producer GUID: {}; Recorded producer GUID: {};"
-              + " Received message producer host: {}; Recorded producer host: {}."
-              + " Multiple leaders are producing. ",
+          "Replica: %s received message at offset %s with upstreamOffset: %d;"
```

Comment:
the new message doesn't contain the string " Multiple leaders are producing. ". Is this on purpose?

Reply:
Good catch! It was a mistake.